pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -204,6 +204,7 @@ backendStdenv.mkDerivation rec {
 
       mv pkg/builds/nsight_systems/target-linux-x64 $out/target-linux-x64
       mv pkg/builds/nsight_systems/host-linux-x64 $out/host-linux-x64
+      rm $out/host-linux-x64/libstdc++.so*
```

Comment:
Hotfix for the regression.
This is very ad hoc and it's OK, we'll keep an open issue to track a proper solution that we can ship later